internal/providersdkv2/data_source_packer_bucket_names.go

```diff
@@ -14,6 +15,8 @@ import (
 	"github.com/hashicorp/terraform-provider-hcp/internal/clients/packerv2"
 )
 
+var defaultPackerTimeout = time.Minute
```

Comment:
I see this was moved here because it was previously defined in one of the removed datasources.

Reply:
yes!